lib/rubocop/cop/rails/safe_nagivation_with_blank.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks to make sure safe navigation isn't used with `blank?` in
+      # a conditional.
+      #
+      # While the safe navigation operator is generally a good idea, when
+      # checking `foo&.blank?` in a conditional, `foo` being `nil` will actually
+      # do the opposite of what the author intends.
+      #
+      # @example
+      #   # bad
+      #   return if foo&.blank?
```

Comment:
I think that `do_something if foo&.blank?` may be preferable to `return if foo&.blank?`. Because it is not specialized for `return`.